### PR TITLE
shed bitfield: Cleanup, add merge command

### PR DESCRIPTION
**Pull request by zgfzgf**
optimize code
use func
add merge

### CODE REVIEW

File: cmd/lotus-shed/bitfield.go [hunk 1]
```diff
@@ -109,43 +81,14 @@ var bitFieldRunsCmd = &cli.Command{
...
 	Name:        "stat",
+	Usage:       "Bitfield stats",
 	Description: "print bitfield stats",
-	Flags: []cli.Flag{
-		&cli.StringFlag{
-			Name:  "enc",
-			Value: "base64",
-			Usage: "specify input encoding to parse",
-		},
-	},
```
Comment (magik6k) [inline cmd/lotus-shed/bitfield.go:119]: I'm pretty sure this flag is needed

Comment (magik6k) [reply cmd/lotus-shed/bitfield.go:119]: (and in other places)

Comment (zgfzgf) [reply cmd/lotus-shed/bitfield.go:119]: super are already enc
use func    Unified use of the upper level enc

Comment (zgfzgf) [reply cmd/lotus-shed/bitfield.go:119]: Name:        "bitfield",
        Usage:       "Bitfield analyze tool",
        Description: "analyze bitfields",
        Flags: []cli.Flag{
                &cli.StringFlag{
                        Name:  "enc",
                        Value: "base64",
                        Usage: "specify input encoding to parse",
                },
        },

**Review by magik6k — APPROVED** [review]
This is nicer, thanks!

**Pull request merged**